securedrop_client/gui/__init__.py

```diff
@@ -176,10 +176,10 @@ def __init__(
         self.setText(text)
         self.elided = True if self.text() != text else False
 
-    def setText(self, text: str) -> None:
+    def setText(self, text: str, wrap_limit: int = 80) -> None:
         self.setTextFormat(Qt.PlainText)
         if self.wordwrap:
-            text = "\n".join(textwrap.wrap(text))
+            text = "\n".join(textwrap.wrap(text, wrap_limit))
```

Comment:
doing this strips away newlines that we want to show in speech bubbles, so i think more work needs to be done in order to make sure we don't strip all newlines from all SecureQLabel. You could add a new bool parameter called `concatenate_text` and use this line of code if it's true, otherwise you could use QLabels `setWordWrap`.